Classes/game.py

```diff
@@ -316,6 +316,33 @@ def setStartRoom(self):
                 self.player.setLocation(room)
                 return
 
+    def addToItemStorage(self, item):
+        """
+        Adds an Item to game storage.
+        """
+        self.itemStorage.append(item)
+
+    def removeFromItemStorage(self, itemName):
+        """
+        Take an item string name and tries to remove that item from storage.
+        Returns the item if successful, False if not.
+        """
+        for item in self.itemStorage:
+            if itemName == item.name:
+                self.itemStorage.remove(item)
+                print(f"Removed {item.name} from storage")
```

Comment:
Can you comment out or remove the print statement?

Reply:
Yep, missed it. Done.